.test-infra/jenkins/CommonJobProperties.groovy

```diff
@@ -163,6 +163,10 @@ class CommonJobProperties {
     // For [BEAM-4847], hardcode Xms and Xmx to reasonable values (2g/4g).
     context.switches("-Dorg.gradle.jvmargs=-Xms2g")
     context.switches("-Dorg.gradle.jvmargs=-Xmx4g")
+
+    // Add docker-pull-licenses option to all Jenkins test to add licenses to when build docker images.
```

Comment:
Wording suggestion:
// Include dependency licences when we build docker images on Jenkins, see: https://s.apache.org/zt68q.

Reply:
I will change the comment with different PR, because this PR triggers 10 tests takes time to run and some tests are flaky.

Reply:
Do you have other comments? If not, I am going to merge it.

Reply:
This is my other comment: I suggested on mailing list that we can also pull licences whenever we run build containers on the release branch. If community does not like that option for some reason, then we should also update the release guide in this PR to make sure release manager is aware that they need to pass a non-default option, and check that licenses were added successfully.

Reply:
I saw on the mailing list that you have updated release guide to include the config flags to build images, and will be updating them. You can do those changes in a follow-up pr.

You can conisider looking at the value of
https://github.com/apache/beam/blob/74a6565c8b64d9fadf256370df47a4c5dadafb55/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L296 

to determine whether the task is running for release branch, instead of asking release manager to pass a config flag.

Reply:
yep, rewording and `isRelease` tag detection for docker image build with be in another PR.